integration_tests/pages/find/courseOffering.ts

```diff
@@ -30,11 +30,15 @@ export default class CourseOfferingPage extends Page {
     })
   }
 
-  shouldHaveOrganisationWithOfferingEmail() {
+  shouldHaveOrganisationWithOfferingEmails() {
     cy.get('.govuk-heading-m').should('have.text', this.organisation.name)
 
     cy.get('.govuk-summary-list').then(summaryListElement => {
       this.shouldContainSummaryListRows(this.organisation.summaryListRows, summaryListElement)
     })
   }
+
+  shouldNotContainSecondaryContactEmailSummaryListItem() {
+    cy.get('.govuk-summary-list').should('not.contain', 'Secondary email address')
+  }
```

Comment:
Does this definitely need testing separately to the above? It feels like the responsibility of `organisationWithOfferingEmailSummaryListRows` to decide what to include as a summary list row, and here we just need to make sure they're all showing up, whatever they are? (See other comment re: `undefined`)